Mathlib/Logic/Embedding/Basic.lean

```diff
@@ -231,14 +231,18 @@ def punit {β : Sort*} (b : β) : PUnit ↪ β :=
 
 /-- Fixing an element `b : β` gives an embedding `α ↪ α × β`. -/
 @[simps]
-def sectl (α : Sort _) {β : Sort _} (b : β) : α ↪ α × β :=
+def sectL (α : Sort _) {β : Sort _} (b : β) : α ↪ α × β :=
```

Comment:
Please add deprecated aliases for `Function.Embedding.sectl_apply` and `Function.Embedding.sectr_apply`.